pkg/cloud-provider/cloudapi/aws/aws_cloudinterface_impl.go

```diff
@@ -105,9 +105,9 @@ func (c *awsCloud) GetAccountStatus(accNamespacedName *types.NamespacedName) (*c
 	return c.cloudCommon.GetStatus(accNamespacedName)
 }
 
-// AddInventoryPoller adds poller for polling cloud inventory.
-func (c *awsCloud) AddInventoryPoller(accountNamespacedName *types.NamespacedName) error {
-	return c.cloudCommon.AddInventoryPoller(accountNamespacedName)
+// DoInventoryPoll adds poller for polling cloud inventory.
+func (c *awsCloud) DoInventoryPoll(accountNamespacedName *types.NamespacedName) error {
+	return c.cloudCommon.DoInventoryPoll(accountNamespacedName)
 }
 
 // DeleteInventoryPoller deletes an existing poller created for polling cloud inventory.
```

Comment:
what about cleanup for deleteInventoryPoller?

Reply:
Removed the functions which are not required.